src/ResourceManager/Sql/Commands.Sql/help/New-AzureRmSqlServer.md

```diff
@@ -39,6 +39,19 @@ This command creates a version 12 Azure SQL Database server.
 
 ## PARAMETERS
 
+### -AssignIdentity
+Generate and assign an Azure Active Directory Identity for this server for use with key management services like Azure KeyVault.```yaml
```

Comment:
@cormacpayne Weird. I used platyPS to generate these. I can fix this, but is this a common issue with platyPS generated help mark down?